Testscripts/Linux/validate-da.sh

```diff
@@ -24,11 +24,17 @@ GetDistro
 
 readonly dir_list=(/opt/microsoft/dependency-agent /var/opt/microsoft/dependency-agent /etc/opt/microsoft/dependency-agent /lib/microsoft-dependency-agent /etc/init.d/microsoft-dependency-agent)
 
-readonly da_pid_file="/etc/opt/microsoft/dependency-agent/config/DA_PID"
+readonly da_installer="InstallDependencyAgent-Linux64.bin"
 readonly uninstaller="/opt/microsoft/dependency-agent/uninstall"
 
+readonly da_pid_file="/etc/opt/microsoft/dependency-agent/config/DA_PID"
+readonly da_log_dir="/var/opt/microsoft/dependency-agent/log"
+readonly da_storage_dir="/var/opt/microsoft/dependency-agent/storage"
+
 function test_case_cleanup(){
-    [ -f "$uninstaller" ] && eval $uninstaller
+    [ -f $uninstaller ] && $uninstaller
```

Comment:
Since there is a need to verify the existence of the file, then doing the `rm` unconditionally doesn't make sense.

```
if [ -f "$uninstaller" ]; then
    $uninstaller
    rm -f "$da_installer"
fi
```

Reply:
Done